### PR TITLE
Add a website in unclassified category

### PR DESCRIPTION
**Pull request by SoulNikhar**
[Wappalyzer](https://www.wappalyzer.com/) is a web technology detection tool that identifies the software and technologies used on websites, providing insights into their technology stack.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -477,6 +477,7 @@ These can be useful for osint and social engineering.
 - [DorkSearch](https://dorksearch.com/) - Speed up your Dorking
 - [usersearch.org](https://usersearch.org/) - Find someone by username or email on Social Networks, Dating Sites, Forums, Crypto Forums, Chat Sites and Blogs
 - [Pastebin](https://pastebin.com/) - Website where you can store text online for a set period of time
+- [Wappalyzer](https://www.wappalyzer.com/) - Wappalyzer is a web technology detection tool that identifies the software and technologies used on websites, providing insights into their technology stack.
```
Comment (edoardottt) [inline README.md:480]: The description should be `Instant access to website technology stacks, company and contact details, social media profiles, email verification and more`

**Review by edoardottt — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (SoulNikhar) [reply README.md:480]: Now description is changed.
Now have a look.

Comment (edoardottt) [reply README.md:480]: I can't see the change

Comment (SoulNikhar) [reply README.md]: Now ??

**Review by edoardottt — APPROVED** [review]
lgtm

**Pull request merged**